phpBB/includes/acp/acp_main.php

```diff
@@ -704,7 +704,7 @@ function main($id, $mode)
 				'S_MBSTRING_ENCODING_TRANSLATION_FAIL'	=> $encoding_translation && ($encoding_translation != 0),
 				'S_MBSTRING_HTTP_INPUT_FAIL'			=> !empty($http_input),
 				'S_MBSTRING_HTTP_OUTPUT_FAIL'			=> !empty($http_output),
-				'S_DEFAULT_CHARSET_FAIL'				=> $default_charset !== 'UTF-8',
+				'S_DEFAULT_CHARSET_FAIL'				=> $default_charset !== 'utf-8',
```

Comment:
```suggestion
				'S_DEFAULT_CHARSET_FAIL'				=> $default_charset !== null && $default_charset !== 'utf-8',
```

Reply:
Did it differently since `$this->php_ini->getString()` can return NULL but `strtolower()` returns a NULL value as an empty string. As per https://www.php.net/manual/en/function.strtolower.php#116230